### ISSUE TITLE
IP range as a text endpoint

### ISSUE THREAD

**Issue by userbradley** [codefresh-io/docs.codefresh.io#419]
Howdy,

I am currently writing a terraform firewall for Digital ocean but have to manually locate your IP range.

Are you able to either:

1. Publish a Terraform Data Source 
2. Provide an endpoint similar to how [cloudflare do](https://www.cloudflare.com/ips-v4) with their list of IP's as a text file 

[Page I came from for context](https://codefresh.io/docs/docs/administration/platform-ip-addresses/)

**Comment by xplrior** [comment]
Text version is location can be found on our documentation page of [Platform IP Addresses](https://codefresh.io/docs/docs/administration/platform-ip-addresses/)

direct link to the text file [here](https://codefresh.io/docs/docs/administration/cf-ip4.txt)

**Comment by userbradley** [comment]
That's awesome, thank!

**Comment by userbradley** [comment]
@xplrior didn't realise you added this as a new feature, thought I missed it.

So let me rephrase it, thanks alot for your hard work on this one! Appreciate it! <3
